apps/frontend/src/app/5_pages/MarketMakingPage/components/BobClaimFeesModal/BobClaimFeesModal.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { FC, useMemo } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  Button,
+  ButtonStyle,
+  ButtonType,
+  Dialog,
+  DialogBody,
+  DialogHeader,
+  SimpleTable,
+  SimpleTableRow,
+} from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { CurrentStatistics } from '../../../../2_molecules/CurrentStatistics/CurrentStatistics';
+import { USD } from '../../../../../constants/currencies';
+import { translations } from '../../../../../locales/i18n';
+import { useGetPool } from '../../hooks/useGetPool';
+import { AmbientPosition } from '../AmbientMarketMaking/AmbientMarketMaking.types';
+import { AmbientLiquidityPool } from '../AmbientMarketMaking/utils/AmbientLiquidityPool';
+import { DEFAULT_SLIPPAGE } from '../BobDepositModal/BobDepositModal.constants';
+import { useGetPoolInfo } from '../BobDepositModal/hooks/useGetPoolInfo';
+import { useGetTokenDecimals } from '../BobWIthdrawModal/hooks/useGetTokenDecimals';
+import { useHandleSubmit } from './hooks/useHandleSubmit';
+
+type BobClaimFeesModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  pool: AmbientLiquidityPool;
+  position: AmbientPosition;
+};
+
+const pageTranslations = translations.bobMarketMakingPage.claimFeesModal;
+
+export const BobClaimFeesModal: FC<BobClaimFeesModalProps> = ({
+  isOpen,
+  onClose,
+  pool,
+  position,
+}) => {
+  const { base, quote } = useMemo(() => pool, [pool]);
+  const { spotPrice } = useGetPoolInfo(pool.base, pool.quote);
+  const { poolTokens } = useGetPool(pool.base, pool.quote);
```

Comment:
```suggestion
  const { spotPrice, poolTokens } = useGetPoolInfo(pool.base, pool.quote);
```
`useGetPoolInfo` calls `useGetPool` under the hood.